hudi-spark-datasource/hudi-spark/src/test/resources/schema_with_logical_types.avsc

```diff
@@ -30,12 +30,11 @@
     },
     {
       "name": "ts",
-      "type": ["long", "null"]
+      "type": ["null", "long"]
     },
     {
       "name": "_hoodie_is_deleted",
-      "type": ["boolean", "null"],
-      "default" : false
```

Comment:
Does this fail any test?

Reply:
Yes, it's usually picked up by higher level stuff, but this test was using the write client directly so it didn't hit the validation. My additions made it fail